UI/MultiIconValueIndicator.cpp

```diff
@@ -28,7 +28,7 @@ MultiIconValueIndicator::MultiIconValueIndicator(GG::X w) :
 
 MultiIconValueIndicator::MultiIconValueIndicator(GG::X w, int object_id,
                                                  const std::vector<std::pair<MeterType, MeterType>>& meter_types) :
-    MultiIconValueIndicator(w, {{object_id}}, meter_types)
+    MultiIconValueIndicator(w, std::vector<int>{object_id}, meter_types)
```

Comment:
This calls the [fill constructor](1) for std::vector. You probably intended to write `std::vector<int>{1, object_id}`.

[1]: http://www.cplusplus.com/reference/vector/vector/vector/